__tests__/components/Form.spec.js

```diff
@@ -405,4 +405,31 @@ describe('Form component', () => {
       component.unmount()
     })
   })
+
+  describe('patternProperties', () => {
```

Comment:
are we concerned with removal of properties in this PR ?

Reply:
Could you expand on your question? Its a little ambiguous

Reply:
looking at the test title `should allow you to add new key value pairs` - would an opposite need to be tested ? e.g. a removal of previously added property key and value by the user of the UI ?

Reply:
@cyplo The functionality for removing key/value pairs exists and works, but isn't tested yet